tiled/_tests/test_xarray.py

```diff
@@ -170,19 +168,17 @@ async def test_serialize_json(ds_node: DatasetAdapter):
 
 
 @pytest.mark.parametrize("ds_node", tree.values(), ids=tree.keys())
+# Suppress warning from tiled.serialization.xarray._BytesIOThatIgnoresClose.close()
+@pytest.mark.filterwarnings("ignore:Exception ignored in")
```

Comment:
@danielballan: Is this an acceptable way to silence the superfluous warnings in this test?  It looks to me like they are related to the overridden behavior of `BytesIO.close()`.